src/user-lists/user-list-events-stats.service.ts

```diff
@@ -443,6 +444,50 @@ export class UserListEventsStatsService {
     return entities;
   }
 
+  async findTopContributorsByProject(options: TopProjectsDto, listId: string): Promise<DbUserListContributorStat[]> {
+    const range = options.range!;
+    const { repo_name } = options;
+
+    const allUsers = await this.findContributorsByType(listId, range);
+
+    if (allUsers.length === 0) {
+      return new Array<DbUserListContributorStat>();
```

Comment:
nit: return an empty array instead of instantiating one.

```suggestion
      return [];
```

Reply:
Addressed this one here and the others in https://github.com/open-sauced/api/pull/489 😎